readme.mdown

```diff
@@ -0,0 +1,62 @@
+# Scrivito Portal App
+
+[![CMS: Scrivito](https://img.shields.io/badge/CMS-Scrivito-brightgreen.svg)](https://scrivito.com) [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT) [![CircleCI](https://dl.circleci.com/status-badge/img/gh/Scrivito/scrivito-portal-app/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/Scrivito/scrivito-portal-app/tree/main) [![GitHub last commit](https://img.shields.io/github/last-commit/Scrivito/scrivito-portal-app.svg)](https://github.com/Scrivito/scrivito-portal-app) [![GitHub top language](https://img.shields.io/github/languages/top/Scrivito/scrivito-portal-app.svg)](https://github.com/Scrivito/scrivito-portal-app)
+
+[Scrivito](https://www.scrivito.com/?utm_source=github&utm_medium=natural&utm_campaign=github_portal_app) is the world's most secure, serverless JavaScript CMS. Based on ReactJS, it is completely maintenance-free, cost-effective, and has unprecedented security. Learn more about Scrivito at [scrivito.com](https://www.scrivito.com/?utm_source=github&utm_medium=natural&utm_campaign=github_portal_app).
+
+This portal app shows Scrivito’s features and is a great basis for your own Scrivito website projects.
+
+## Features
+
+Scrivito is a SaaS Content Management Service, built for digital agencies and medium to large sized businesses.
+
+It is the only CMS that combines unmatched WYSIWYG usability for editors with maximum flexibility for developers.
+
+And because Scrivito is 100% serverless, you'll never have to patch your CMS, so it's the most secure and cost-effective solution on the market.
+
+## Getting Started
+
+### Clone repository
+
+```
+git clone https://github.com/Scrivito/scrivito-portal-app.git
+
+cd scrivito-portal-app/
+```
+
+### Install/check prerequisites
+
+Make sure that you have [![node ≥ 20.0](https://img.shields.io/badge/node-≥%2020.0-blue.svg)](https://nodejs.org) installed.
+
+```
+node -v
+# should output something like v20.17.0
+```
+
+### Install dependencies
+
+Now call `npm install` in the command line to install all JS dependencies (which are placed in the folder `node_modules/`).
```

Comment:
* [ ] FYI since this app is geared towards experienced (TypeScript) developers the parenthesized part could be left out. 
* [ ] FYI "all JS dependencies". Actually it's also TS dependencies. Leave out?

Reply:
I would keep it as is.